### ISSUE TITLE
Vaidator profile page design

### ISSUE THREAD

**Issue by serejandmyself** [citizenweb3/validatorinfo#28]
### Top part visual (Separate task (non demo related): #39) 
- Each validator profile page is presented with a visual of the connected nodes (map of zones style)
- The visual can be expanded to full screen view

### Top part of the page (stays the same as we scroll through tabs) 
- Clear name at the top, after the words `Validator Profile:`
- Blocks: (A) validator badges, (B) playable player with our interview if applicable (https://serejandmyself.github.io/ (an example on my blog), (C) block with short description and a button to expand social links (Website link is visible with a clickable icon outside the links) - the button itself needs to attract attention to it (D) Claimed glow button if the page is claimed by a validator 
- Claimed (by default it doesnt show up, but we want to show an example with claimed) is actually a badge! So claimed goes together with badges on one side, and info with links and player on the other side
- Example of social links drop down menu: https://suiscan.xyz/mainnet/validator/0x61953ea72709eed72f4441dd944eec49a11b4acabfc8e04015e89c63be81b6ab/delegators (its also a validator page analogue)

### Tabs
1) **Network table** (open by default) 
- Table with networks a validator validates, an example: https://polkachu.com/validator_entities/82DA4FF029A26E24
- Example: (cosmostations validators networks - second bottom of page): https://www.mintscan.io/visualization/validators/cosmostation
- Containing the following metrics: network, validator rank on it, voting power, commission, self delegation, total delegetions, missed blocks, fee, uptime, participation rate, fans (amount of delegators), APY, status (active/not - its a red, green or yellow circle)
- Contains pie chart visual of stake distribution on given network
- When clicking on a network in that table it takes us to: /validators/X/cosmoshub NOT /networks/cosmohub

2) **Metrics**
- Technical score, Social score, Governance score, User score (TS, SS, GS, US) from main table shown here as 4 separate blocks on top with an arrow button that opens a drop down box with information
- Boxes with TVS, Fans, Assets underneath the top boxes with numbers
- A graph/chart of how the TVL and the total stakers grow over time. 
- An example: https://www.stakingrewards.com/provider/luganodes?category=pos&sort=staking_wallets&timeframe=30d&order=desc (no need for the info on the left ((aum/stakers))

3) **Revenue** 
- Shows revenue of the validator for the end user: this is a subjective(!) tab
- Some validators drop tokens to users. Example: https://cosmospug.com/airdrops/?validator_specific=yes
- There should be an info box that shows if there is an own token that the validator has (something similar to the example above)
- There should be a clear visualization of 2 icon/logo: (A) re-stake icon and info (possibly another drop down menu), (B) a badge in case validator has slashing protection
- A box that shows a validators Daily commission by network. Example: https://inbloc.org/cosmos/validator/cosmosvaloper1tflk30mq5vgqjdly92kkhhq3raev2hnz6eete3 (daily commission is presented by network here as a text line. we can present it as a `metric box` here by token)
- Just like it is presented now where you have each token in black, there will be a clickable menu of the token the validator has. Once a user clicks on a token, i.e. `juno` - A form with a calculator will appear below. Something like this, but more simple: https://www.stakingrewards.com/asset/solana (where it says `calculate your sol staking amount` ) - middle of the page, middle calculator and form, where the user enters `amount of tokens they staked` and `time period`. That form will take into account: the APY of the validator over the given period of time (minus any slashing events) (minus commission of validator) + validator specif airdrops, such as the case of posthuman. And the calculator will return an answer for 1, 7, 30, 365 days in usd / or token. 
- Growth against network: on the left a chart is drawn (like with the calculator) which would show 2 lines, one the returns with the validator and another one, in different color -taking into account average apy, commission and average staking rewards, and compare them over time 
- Staking history: https://inbloc.org/validator/cosmosvaloper1e859xaue4k2jzqw20cv6l7p3tmc378pc3k8g2u (staking stats history) (changes as the user picks a different token - the whole screen will change ((only the form of the calculator will remain with `__` for input))

4) **Public Good**
- A tab that will clearly present everything a validator offers as public good, for example this can be a podcast they have, it can be some infrastructure they are running for the network apart from validator nodes, etc
- The tab should have blocks: (A) INFRASTRUCTURE (B) MEDIA (C) TOOLS (D) COMMUNITY (E) OTHER 
- The info in this tab will mainly rely on user additions - they will be able to login and submit something to those categories by signing a tx. But we will also submit to here
- The infra section is a table of nodes: rpc nodes, relayer nodes, archive nodes, other nodes
- Additions can also be known interviews to podcasts or on conferences. It can be guides a validator wrote, an applications they created, something they organized, etc  
- Anything a user will add / approve / disprove will give the user a badge / score?

5) **Governance** 
- Validator governance participation. Validators vote on proposals on networks. This is an important part of a networks life. A simple idea for a visualization is a news feed, with latest votes on latest props. We can strive to make this a highlighted feature, as of today this remains a very high demand feature, thats not been implemented anywhere 
- Not to mix with the governance tab on `validator X on network Y` page. This one is a summary, feed. Maybe some visuals. The other one is detailed voting history of a validator on a given network
- Example (voting activity): https://www.mintscan.io/cosmos/validators/cosmosvaloper1clpqr4nrk4khgkxj78fcwwh6dl3uw4epsluffn - but this is more about the second case (a table with stats)
- We should be able to see the past proposals and their success rate (percentage and vote visual) - the ones that the validators created
- Example of participation rate visualization: https://www.mintscan.io/analytics/governance/trends
- Idea: pie charts and the colors of the different votes: https://inbloc.org/cosmos/governance (here it is used to summarize a network view on a proposal)
- An important feature to design: voting filters (who voted the same, voting history) - wont work now, but will be used later. Basically if a user clicks on this button, they should be shown other valdiators with similar votes. It doesnt make sense to compare vote by vote here. More along the lines of "show me validators that have similar opinions" - this will check the voting history and find other with the same one

**Comment by serejandmyself** [comment]
Main elements implemented. Fixes to follow in a separate task
